app/views/welcome/_chat.html.erb

```diff
@@ -10,9 +10,17 @@
       <span class="chat-receiver" data-chat-id=<%= chat.id %>>
         <%= chat.receiver_id == current_user_id ? (chat.sender_name) : (chat.receiver_name)%>
       </span>
-      <span class="chat-topic"
-        <i><%= image_tag("#{chat.topic_id}.png")  %></i>
-      </span>
+      <div class="chat-topic">
+        <i class= "chat-topic-img"><%= image_tag("#{chat.topic_id}.png")  %></i>
+        <% if results[chat.id] > 0 %>
+            <%= render 'unread_msg', chat: chat, results: results %>
+        <% end %>
+        <div id= "<%= chat.id%>" class= "unread-msg-div display-none" data-unread-count= <%= results[chat.id]%>>
```

Comment:
Not sure if it´s a good idea to have an element id called just with a number. Maybe something like id= "chat-<%= chat.id%>"